src/pages/mentees.tsx

```diff
@@ -91,16 +100,35 @@ function MenteeTable({ users, refetch }: {
       </Tr>
     </Thead>
     <Tbody>
-      {users.map((u: any) =>
-        <MenteeRow key={u.id} user={u} refetch={refetch} />)
+      {users.sort((a: User, b: User) => {
+        const idA = Number(menteeToYear[a.id]);
```

Comment:
Don't convert to Number because the strings may not necessarily be numbers. Simply use `localeCompare` to compare the two strings.